packages/components/src/components/wallet-cfd-success-dialog/wallet-cfd-card.scss

```diff
@@ -0,0 +1,78 @@
+.wallet-cfd-card {
+    width: 24rem;
+    height: 14.4rem;
+    position: relative;
+    overflow: hidden;
+    border-radius: $BORDER-RADIUS;
+    background: var(--wallets-card-active-gradient-background);
+
+    @include mobile {
+        width: 21.6rem;
+        height: 12.8rem;
+    }
+
+    &__icon {
+        position: absolute;
+        top: 1.6rem;
+        left: 1.6rem;
+    }
+
+    &__cfd-icon {
+        position: relative;
+        z-index: 1;
+    }
+
+    &__wallet-icon {
+        position: absolute;
+        top: 1.2rem;
+        left: 1.2rem;
+        z-index: 0;
+    }
+
+    &__badge {
```

Comment:
Try to use Badge component inside of the tsx file with `rounded_corners={2}` instead of this